pkgs/os-specific/linux/syslinux/default.nix

```diff
@@ -1,14 +1,15 @@
-{ stdenv, fetchFromRepoOrCz, fetchurl, nasm, perl, python3, libuuid, mtools, makeWrapper }:
+{ stdenv, fetchgit, fetchurl, fetchpatch, nasm, perl, python3, libuuid, mtools, makeWrapper }:
 
 stdenv.mkDerivation {
   name = "syslinux-2019-02-07";
```

Comment:
Please split this into pname and version.

Reply:
done